EasyPost/Billing.cs

```diff
@@ -0,0 +1,116 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using RestSharp;
+
+namespace EasyPost
+{
+    public static class Billing
+    {
+        /// <summary>
+        ///     Delete a payment method.
+        /// </summary>
+        /// <param name="priority">Which type of payment method to delete.</param>
+        /// <returns>Whether the request was successful or not.</returns>
+        public static async Task<bool> DeletePaymentMethod(PaymentMethod.Priority priority)
+        {
+            PaymentMethodObject paymentMethod = await GetPaymentMethodByPriority(priority);
+
+            return await DeletePaymentMethod(paymentMethod);
+        }
+
+        /// <summary>
+        ///     Delete a payment method.
+        /// </summary>
+        /// <param name="paymentMethodObject">Payment method to delete.</param>
+        /// <returns>Whether the request was successful or not.</returns>
+        public static async Task<bool> DeletePaymentMethod(PaymentMethodObject paymentMethodObject)
+        {
+            Request request = new Request($"{paymentMethodObject.Endpoint}/{paymentMethodObject.id}", Method.Delete);
+
+            return await request.Execute();
+        }
+
+        /// <summary>
+        ///     Fund your wallet from a specific payment method.
+        /// </summary>
+        /// <param name="amount">Amount to fund.</param>
+        /// <param name="priority">Which type of payment method to use to fund the wallet.</param>
+        /// <returns>True if successful, false otherwise.</returns>
+        public static async Task<bool> FundWallet(string amount, PaymentMethod.Priority priority)
+        {
+            PaymentMethodObject paymentMethod = await GetPaymentMethodByPriority(priority);
+
+            return await FundWallet(amount, paymentMethod);
+        }
+
+        /// <summary>
+        ///     Fund your wallet from a specific payment method.
+        /// </summary>
+        /// <param name="amount">Amount to fund.</param>
+        /// <param name="paymentMethodObject">Payment method to use to fund the wallet.</param>
+        /// <returns>True if successful, false otherwise.</returns>
+        public static async Task<bool> FundWallet(string amount, PaymentMethodObject paymentMethodObject)
```

Comment:
Issue: We should be abstracting away the need for the user to pass us a whole payment method. Instead, this function should accept `primaryOrSecondary` and adjust the endpoint and logic accordingly.

Reply:
Removed